kong/include/kong/services/config/v1/config.proto

```diff
@@ -100,7 +100,7 @@ message SyncConfigRequest {
   //
   // DP MUST NOT combine configuration from two SyncConfigRequest. Config in
   // each request is self-contained.
-  kong.model.Config config = 1;
+  bytes config = 1;
```

Comment:
It is a good simplification.

Reply:
let's also rm -rf `kong/include/kong/model/`